acceptance/assertions/image.go

```diff
@@ -69,6 +69,15 @@ func (a ImageAssertionManager) HasLabelWithData(image, label, data string) {
 	a.assert.Contains(label, data)
 }
 
+func (a ImageAssertionManager) HasLabelWithNoData(image, label, data string) {
```

Comment:
```suggestion
func (a ImageAssertionManager) HasLabelNotContaining(image, label, data string) {
```

Maybe we could change the above `HasLabelWithData` -> `HasLabelContaining` for parity. `HasLabelWithNoData` makes me think that we're asserting the label is completely empty.